src/files.ts

```diff
@@ -25,13 +25,18 @@ export class Files {
    * @param file Can be a:
    * - local or temp filepath such as /path/to/local/file.pdf;
    * - base64 image string such as data:@file/pdf;base64,....
+   * @param inbox_id Unique identifier for the inbox to which the file belongs
    * @returns {PredictedFileOutput} The prediction results
    */
-  async post(file: string): Promise<PredictedFileOutput> {
+  async post(file: string, inbox_id: string): Promise<PredictedFileOutput> {
     /** Throw an error if the user submits a file that is not in the scope */
     if (!file || typeof file !== 'string') {
       throw new Error('Please provide the file in a valid string');
     }
+    /** Throw an error if the user submits without indicating the inbox_id */
+    if (typeof inbox_id !== 'string') {
+      throw new Error('Please provide the correct inbox_id');
```

Comment:
Maybe we can provide some additional errors on length of the string (should be 24 characters), and write in the error more specifically what is wrong; as here 'please provide a string for inbox_id'

Reply:
added this just below that exception
    /** Throw an error if the user submits the inbox_id but it has the wrong length */
    if (inbox_id.length && inbox_id.length !== 24) {
      throw new Error('Please provide a correct inbox_id');
    }